containers/agent/deploy.go

```diff
@@ -41,13 +49,29 @@ var (
 		tags           string
 		skipTags       string
 	}
+
+	openstackConfigVersionGVR = schema.GroupVersionResource{
+		Group:    "osp-director.openstack.org",
+		Version:  "v1beta1",
+		Resource: "openstackconfigversions",
+	}
+
+	filteredAllNodesConfig = []string{
+		"oslo_messaging_notify_short_bootstrap_node_name",
+		"oslo_messaging_notify_node_names",
+		"oslo_messaging_rpc_node_names",
+		"memcached_node_ips",
+		"ovn_dbs_vip",
+		"redis_vip",
```

Comment:
wondering if this should be a parameter of the osdeploy (defaulting to this list) and then passed into the agent, or the agent get it from checking the osdeploy CR? with that this list could be easy customized in case filter requirements change.

Also filter is only done for a cell export [1] from the central controller deployment stack. In case of splitting the cell controllers and cell computes in different stacks, you use the same export like DCN, where there is no filtering [2], also not for DCN export [3]. So like we discussed in one of our previous meetings, we probably should just do both exports, one which filters and the one without?

[1] https://github.com/openstack/python-tripleoclient/blob/stable/train/tripleoclient/v1/overcloud_cell.py#L89
[2] https://github.com/openstack/python-tripleoclient/blob/stable/train/tripleoclient/v1/overcloud_cell.py#L92
[3] https://github.com/openstack/python-tripleoclient/blob/stable/train/tripleoclient/v1/overcloud_export.py#L93

Reply:
@stuggi just pushed a commit which splits out both filtered and non-filtered version of the file in the ConfigMap. Still need to add documentation on this to the README.
On the idea of making it a parameter while we could do that I suppose I'd rather not. Tripleclient has these as a constant so I'm not sure they change that often.